src/components/dialogs/parameters/voltageinit/voltage-limits-parameters.js

```diff
@@ -111,7 +114,10 @@ const VoltageLimitsParameters = () => {
             },
         ].map((column) => ({
             ...column,
-            label: intl.formatMessage({ id: column.label }).toUpperCase(),
+            label: intl
```

Comment:
can you create an util function for this part? you are using it 3 times, the same way